src/controllers/connectionConfig.ts

```diff
@@ -0,0 +1,67 @@
+import { rejects } from "assert";
+import { resolve } from "path";
+
+const Connection = require('tedious').Connection;
+const RequestStatement = require('tedious').Request;
+let connection;
+
+export function setupConnection(portVal){

```

Comment:
I'm sure this is just something you've left in from an early attempt, but you shouldn't need to pass the port that express is hosted on around (it's not used in this method, but more generally should only be used to configure and expose your express application);